migrate.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 
-heroku run conjurctl db migrate
+heroku run possum db migrate
```

Comment:
This is wrong -- it's what broke the eval.conjur.org deploy last week. The command should be `conjurctl`.